includes/fbutils.php

```diff
@@ -429,6 +429,52 @@ public function get_variant_option_name( $wp_id, $label, $default_value ) {
 			return $term && $term->name ? $term->name : $default_value;
 		}
 
+		/**
+		 * Get all products for synchronization tasks.
+		 *
+		 * @return array IDs of all product for synchronization.
```

Comment:
I think it's worth adding a warning here to future devs. Something like "If you make any changes to this method please check it will still scale for stores with a massive number of products."